boards/arm/adafruit_qt_py_rp2040/seeed_xiao_connector.dtsi

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2023 Ian Wakely
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	xiao_d: connector {
+		compatible = "seeed,xiao-gpio";
+		#gpio-cells = <2>;
+		gpio-map-mask = <0xffffffff 0xffffffc0>;
+		gpio-map-pass-thru = <0 0x3f>;
+		gpio-map
+			= <0 0 &gpio0 29 0>		/* D0 */
```

Comment:
Any particular reason this is formatted the way it is? Makes more sense to me to have the commas after the value like it does here:

https://github.com/zephyrproject-rtos/zephyr/blob/8c83185f6e3d5b41ea86aa01bccd82fc07cb1de5/boards/arm/nucleo_f401re/arduino_r3_connector.dtsi#L13-L16

Reply:
I copied this file from another board that must have had it that way. I was more focused on making sure that the pins were mapped correctly and missed the weird commas. I'll move the commas.